src/App.css

```diff
@@ -0,0 +1,119 @@
+/*Its a simplified version of Dracula*/
```

Comment:
just fyi... Libraries such as Mui, tailwind, bootstrap also exist if you want to avoid doing a lot of CSS